httpbin/handlers_test.go

```diff
@@ -2897,10 +2935,10 @@ func TestHostname(t *testing.T) {
 
 		realHostname := "real-hostname"
 		app := New(WithHostname(realHostname))
-		srv, client := newTestServer(app)
-		defer srv.Close()
+		env := newTestEnvironment(app)
+		defer env.srv.Close()
 
-		req, err := http.NewRequest("GET", srv.URL+"/hostname", nil)
+		req, err := http.NewRequest("GET", env.srv.URL+"/hostname", nil)
```

Comment:
I don't think these changes are needed?

Reply:
Ah, maybe they're necessary because `newTestServer` was deleted?  I guess we could keep it around, and make `newTestEnvironment` use it?

Reply:
Actually I replaced newTestServer with newTestEnvironment. Because there were only one call remaining which might use direct variables I changed it. But I changed it back.